README.md

```diff
@@ -2,31 +2,160 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Initialize an array with object references
+- Iterate over an array of objects
 
-## Introduction
+## Arrays of objects
 
-A 1-2 sentence summary of what will be covered.
+An array can store primitive types (`int`, `boolean`, etc.) or it can
+store reference types. For example, an array of `String` objects can be declared
+and initialized as shown: 
 
-## Topic 1
+```java
+String[] names = {"Leslie", "Ann", "Ron", "Ben"};
```

Comment:
Yass! Parks and Rec! 

[insert gif on Parks and Rec](https://media.giphy.com/media/zyin7TYoGmLAs/giphy.gif)